anchore_engine/analyzers/syft.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import json
+import collections
+
+import anchore_engine.analyzers.utils
+from anchore_engine.clients.syft_wrapper import run_syft
+
+
+def handle_gem(findings, artifact):
+    pkg_key = artifact['locations'][0]['path']
+
+    # craft the artifact document
+    pkg_value = {
+            'name': artifact['name'],
+            'versions': [artifact['version']],
+            'latest': artifact['version'],
+            'sourcepkg': artifact['metadata'].get('homepage', ''),
+            'files': artifact['metadata'].get('files', []) or [],
```

Comment:
At first sight it looks redundant to say `or []` because the `.get()` is falling back to `[]` when `'files'` is not present. But, is it possible this is accounting for a situation where `'files'` is a `None` or some other value and this wants to enforce an empty list if so?

Reply:
That's right, with syft v0.2.0 there were fields that were explicitly `null`, so the `get()` would positively get `None` value back. Since this was fixed in v0.3.0 at the end of last week I can remove this logic.